js/measurements.js

```diff
@@ -15,7 +15,7 @@ for (unit of UNITS) {
 select.onchange = function() {
     for (measurement of elem_measurements.getElementsByClassName('measurement')) {
         var multiplier = MULTIPLIERS[UNITS.indexOf(select.value)];
-        measurement.textContent = measurement.attributes.magnitude.value * multiplier;
+        measurement.textContent = Math.floor(measurement.attributes.magnitude.value * multiplier * 100)/100;
```

Comment:
Good move, should have done this from the beginning. That said, `.toFixed(2)` would be a cleaner solution to this.

Reply:
Thanks for the tip, I have implemented it!